src/actions/users.js

```diff
@@ -236,6 +237,7 @@ export function loadMe(): ActionFunc {
             getMyTeams()(dispatch, getState),
             getMyTeamMembers()(dispatch, getState),
             getMyTeamUnreads()(dispatch, getState),
+            getMyTermsOfServiceStatus()(dispatch, getState),
```

Comment:
please change this to `dispatch(getMyTermsOfServiceStatus())` and while doing that if you want to change the others that would be great